### ISSUE TITLE
VR&E Landing page layout 

### ISSUE THREAD

**Issue by emilyville** [department-of-veterans-affairs/vets-website#2161]
Some layout things to fix pre-launch

- The blue boxes should span the full page - as it is now they are not full width.
- The copy above the blue boxes is too small. It should look like the intro text on this page: https://www.vets.gov/education/
- We don't intend to use piano keys and blue boxes on the same page. Also there's only one piano key with no sub-heading so it just looks like words on the page (to me) and looks odd above the apply for button bar... I suggest changing it to a 2-up blue cards and making this the fourth card. One day we will have a better way to call out other audiences, but we don't yet. 

Tagging @mariana-almeida and @gnakm for thoughts on that last thing. When we're all in agreement we'll assign to @kantrowitz !

![screencapture-vets-website-pr-1801-herokuapp-com-vre-1461932964614](https://cloud.githubusercontent.com/assets/1307774/14915941/5f13091e-0dcb-11e6-9678-666303c7bf50.png)

**Comment by emilyville** [comment]
In addition.. Now that I'm looking at it I see we address different audiences with each card, as well. It would make sense to format the title for the fourth blue card to be consistent with the other two, like so:

Service Disabled
sub description

Family Members
sub description

Servicemembers
sub description

Employers
sub description

Also tagging @maryannbrody for awareness - we had a long conversation about asking users to identify into categories.

**Comment by kantrowitz** [comment]
@rcarthur

**Comment by mariana-almeida** [comment]
@emilyville suggested moving Employers to blue card. I think we should instead remove it from this page. This page is pretty deep in the IA, and employers will have by now already navigated to their own area. We had nearly removed it in an earlier iteration of this.

**Comment by gnakm** [comment]
Design wise, everything makes sense (Employers becoming a blue card).

I thought a lot about this (mostly because I'm not an expert in the VR&E topic) but I also agree that we should scrap it because if we follow a employer's path, it will bring them through a loop if we leave the employers card here. It would literally redirect them back to the same page.

**Comment by maryannbrody** [comment]
Jumping in since @emilyville tagged me. I agree with @gnakm and @mariana-almeida that it makes sense to just remove the employer page. 

As far as the audience thing goes, generally it doesn't test well when you split things up by audience and expect people to make a choice -- plus you can end up with duplicative content which is not necessary. For example, this page: http://www.ptsd.va.gov/professional/index.asp -- You have "For the public" and "for professionals" (which you see a lot). If you click -- you'll see there's lots of duplicative content since it's important for both of those audiences. 

A better way to write the PTSD content would be something that is labeled around the specific action that those users will want to do (since people can talk about themselves differently-- like someone who is still active duty-- they would not identify as "the public"). Something like "Learn more about/Get help with PTSD" and "Get clinical training materials to treat PTSD".

I'm walking in totally cold here, so I'm not sure where this work stands but -- can this just be one page with sub-headers for each audience and a big button at the bottom to apply?  It adds a lot to make someone make choices and click. ("I just had back surgery and my medical discharge paperwork is in motion, but i'm still active duty. Am I a service member or am I service-disabled?")

So, a page that answers:
- What is the program
- Who is eligible
- What can each group of people get
- How do i apply

**Comment by mariana-almeida** [comment]
@maryannbrody 
Well said, on the audience issue. It's generally fraught with problems. In this case, it happens to be the lesser of two evils design-wise. 

Re: "can this just be one page with sub-headers for each audience and a big button at the bottom to apply" - I know where you're going with this. 

The problem is that we have 6 distinct versions of the program (5 for veterans, 1 for family) each with different eligibility, completely distinct program specifics. So it becomes unwieldy to put that many on one page. 

In short: I'm comfortable leaving this as is (more or less) until we can redesign the Level 2 page, and the use that for this content. 

I'm still in favor of scrapping the employers from this page - again because this info is already part of the Careers and Employment Employer side, and the placement in the navigation is pretty down deep in Job Seekers where employers won't find it. 

Tagging @kantrowitz to remove Employers. @emilyville feel free to re-open if you disagree.

**Comment by emilyville** [comment]
pinging @kantrowitz for the layout change and the removing the lone piano key

**Comment by emilyville** [comment]
Thank you for removing the lone piano key @kantrowitz ! 

The cards need to be full width and the text on top should be bigger and blue like this guy: https://www.vets.gov/education/

then all good.
